product_state/views/product_template.xml

```diff
@@ -57,6 +57,8 @@
                     </group>
                     <group string="Description">
                         <field name="description" nolabel="1" />
+                        <field name="default" widget="boolean_toggle" />
+                        <field name="active" widget="boolean_toggle" />
                     </group>
```

Comment:
pls add a fix on line 26
` <field name="domain">[('id', '=', product_state_id)]</field>`

Reply:
Thanks for the advice, it has been fixed.